src/ComposerRequireChecker/Cli/CheckCommand.php

```diff
@@ -134,6 +135,15 @@ private function getCheckOptions(InputInterface $input): Options
         if (!$fileName) {
             return new Options();
         }
+
+        if (Phar::running() !== '') {
+            $fileName = realpath($fileName);
+            if (false === $fileName) {
+                throw new \InvalidArgumentException('file not found: [' . $input->getOption('config-file') . ']');
+            }
```

Comment:
This exception message is a bit too generic

Reply:
what would you suggest? something like this?
```
could not find given config-file [...]
```